src/app/common/config/abi.ts

```diff
@@ -0,0 +1,5321 @@
+import { NETWORKS } from '~lib/utils/envHelper';
+
+export const ABI_VERSION = {
+   setterContract: {
+       [NETWORKS.MAINNET]: [
```

Comment:
It is not enough to use only mainnet/prater for getting abi object.
On production there will be 3 networks in a toggle: mainnet, testnet jato, and testnet jato 2.
Testnet jato and jato 2 are both prater, but will have different ABI structure.
So you need to get ABI here per `${version}_${network}`

Reply:
done